sourced/ml/utils/__init__.py

```diff
@@ -1,4 +1,5 @@
 from sourced.ml.utils.bigartm import install_bigartm
+from sourced.ml.utils.bblfsh_roles import IDENTIFIER, QUALIFIED, LITERAL
```

Comment:
wow. 
Can we avoid doing like that?
It is hard to understand what it is about if you see in code just alone `LITERAL`.

Usually I prefer to import like:
```python 
from sourced.ml.utils import bblfsh_roles

# And use in code like: 
bblfsh_roles.LITERAL #  Here you see what it is exactly.
```

Reply:
^